src/MunicipalityRegistry.Projections.Legacy/MunicipalitySyndication/MunicipalitySyndication.cs

```diff
@@ -45,6 +45,7 @@ public Instant LastChangedOn
         public Organisation? Organisation { get; set; }
         public string? Reason { get; set; }
         public string? EventDataAsXml { get; set; }
+        public DateTimeOffset SyndicationItemCreatedAt { get; set; }
```

Comment:
Can you tell me a bit about why `RecordCreatedAt` did not work?

Reply:
RecordCreatedAt is CRAB record related. It has no relation to when a record in Syndication was created and can have a completely different time.

Reply:
It is the timestamp the object was first created.

Reply:
Ok, makes sense